opctl/opctl
# upgrade deps to fix some ops that are incompatible with older versions of docker

# Overview
Some folks noticed that a private op used at Remitly broke recently when they upgraded their docker client. This change resolves that issue by upgrading the version of the docker library that opctl uses. I'm keeping this PR as a draft right now because I may try to split it up into smaller dependency updates to make sure that none of these updates breaks opctl

## Testing
1. build a binary with these changes
2. run the previously broken op using the newly built binary
3. run some other ops that have always been working to confirm that they haven't been broken
